pyscripts/build_docker_image.py

```diff
@@ -121,34 +130,37 @@ def build_image(tags):
         cache_from=cache_from,
         dockerfile=dockerfile,
     ):
-        try:
-            sys.stdout.write(json.loads(line.decode("utf-8"))["stream"])
-        except KeyError:
```

Comment:
Can you please add this back in?  I'm not sure why this was removed.

Reply:
Fixed